src/test/java/com/google/api/codegen/testsrc/common/library_gapic.yaml

```diff
@@ -129,12 +129,6 @@ interfaces:
     flattening:
       groups:
       - parameters: []
-    page_streaming:
```

Comment:
This method doesn't follow the OnePlatform standards on paged methods, so it wouldn't count as a paged method if we solely looked at the protofile.